simplecpp.cpp

```diff
@@ -692,7 +690,7 @@ void simplecpp::TokenList::readfile(Stream &stream, const std::string &filename,
             continue;
         }
 
-        if (std::isspace(ch)) {
+        if (ch <= ' ') {
```

Comment:
hmm I am not sure. The compiler knows the ascii codes. But if we assume the developer does not.. isn't it more explicit that '\t' '\n' '\r' are handled using std::isspace(ch) than `ch <= ' '`.

Reply:
We basically separate the ASCII table is three parts with our code:

`>= 0x80` - non-ASCII characters we bail out on

`< ' '` - whitespaces and control characters which we treat as allowed and converted to `' '`

... and the printable ASCII characters

So we only have either `' '` or a printable ASCII so the only character `isspace()` can match is `' '`. So we get rid of the conversion step and simply exclude the lower part of the table.

Reply:
but with these changes you assume that `< ' '` will match `\t` and `\r` and `\n` ... 

on the other hand we did assume that `< ' '` would match all other control characters already.

I don't have a strong opinion about this .. just wanted to point out that there might be a good reason to be explicit.

Reply:
> I don't have a strong negative opinion about this .. just wanted to point out that there might be a good reason to be explicit

I think the code is explicit enough since we are limiting us to the printable ASCII characters. If we weren't `ìsspace()` would have been necessary.